source/help/getting-started/messaging-basics.rst

```diff
@@ -0,0 +1,45 @@
+Messaging Basics
+================
+
+--------------
+
+**Write messages** using the text input box at the bottom of Mattermost.
+Press **ENTER** to send a message. Use **Shift+ENTER** to create a new
```

Comment:
Another thing is whether to have spaces between the keyboard button labels, e.g. `SHIFT+ENTER` or `SHIFT + ENTER`.

Again, we're pretty inconsistent.